lms/static/js/leanModal.js

```diff
@@ -0,0 +1,136 @@
+(function($) {  // eslint-disable-line wrap-iife
```

Comment:
This is not as straightforward as I expected it to be. 
Bootstrap CSS is not included by default into main Django template and I can't find a way to reference it from within the project. I can reference externally hosted version but it breaks the existing styling.
We can file a tech debt ticket since this is nice-to-have.

Let's move with making it work and any nice-to-have's can be a separate task.